packages/vitest/config.d.ts

```diff
@@ -1 +1,3 @@
+// ensure `@vitest/expect` provides `chai` types
+import type {} from '@vitest/expect'
```

Comment:
I initially tried adding this in https://github.com/vitest-dev/vitest/blob/82ee70b91fc233b4c05980a4c1dbaac41f0fec67/packages/vitest/src/integrations/chai/config.ts#L1 but it looks like dts emit drops `@vitest/expect` import.